client/src/components/SearchPage/SearchField.js

```diff
@@ -0,0 +1,111 @@
+import React, {Component} from 'react';
+import './SearchField.css'
+import Logo from '../assets/logo.png'
+import Search from '../assets/search.png';
+import Mic from '../assets/mic.png'
+import Suggestion from "./data";
+import data from './data'
+class SearchField extends Component{
+
+
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            words: [],
+            userInput: '',
+            matches: [],
+            rowHighlighted: -1
+        }
+    }
+
+    componentDidMount(){
+        this.setState({
+            words: data.words
+        })
+    }
+
+    handleUserInput(e) {
+        let input = e.target.value;
+        let {words} = this.state;
+        let matches = [];
+
+        if (input){
+            for (let i = 0; i < words.length; i++){
+                if (words[i].startsWith(input) && matches.length < 10){
+                    matches.push(words[i])
+                }
+            }
+        }
+
+        this.setState({
+            userInput: input,
+            matches: matches
+        })
+    }
+
+    selectAutocomplete(i){
+        this.setState({
+            userInput: this.state.matches[i],
+            matches: []
+        })
+    }
+
+    handleKeyPress(e){
+        let {rowHighlighted} = this.state;
+        if (e.key === 'ArrowUp' && rowHighlighted > -1){
+            rowHighlighted--;
+        }
+        if (e.key === 'ArrowDown' && rowHighlighted < this.state.matches.length - 1){
+            rowHighlighted++;
+        }
+        if (e.key === 'Enter'){
+            return this.selectAutocomplete(rowHighlighted);
+        }
+
+        this.setState({rowHighlighted});
+    }
+
+    setRowHighlighted(i){
+        this.setState({
+            rowHighlighted: i
+        })
+    }
+
+
+    render() {
+        return(
+            <div>
```

Comment:
Again. Bad code style. Don't indent your code poorly. Just use a linter (Prettier) if you're not sure what kind of style to follow.